assets/js/commento.js

```diff
@@ -215,10 +226,7 @@
 
 
     var _getComments = function() {
-        var data = {
-            "url": document.location
-        };
-        post(_api.get, data, function(reply) {
+        get(_api.get + '?url=' + document.location, function(reply) {
```

Comment:
@adtac, using http.Request.URL.Query().Get("foo") will de-encode automatically, for example:

```go
url := r.URL.Query().Get("url")
uraw := r.URL.RawQuery
Logger.Infof("URL: %s", url)
Logger.Infof("Raw: %s", uraw)
```

output:
```
2017/06/19 13:08:19 URL: file:///Users/stuart/Desktop/commento.html?test encoding
2017/06/19 13:08:19 Raw: url=file:///Users/stuart/Desktop/commento.html?test%20encoding
```